packages/examples/cvat/exchange-oracle/src/utils/annotations.py

```diff
@@ -234,7 +236,7 @@ def __init__(
                 dst_label_cat = dm.LabelCategories(attributes=deepcopy(src_label_cat.attributes))
 
                 for dst_label in dst_labels:
-                    assert isinstance(dst_label, str) or isinstance(dst_label, tuple)
+                    assert isinstance(dst_label, str | tuple)
```

Comment:
```suggestion
                    assert isinstance(dst_label, (str, tuple))
```

Reply:
Sorry, I don't see clear basis for this suggestion. This was not changed by accident, if that is what you're implying.

Since Python 3.10, UnionType can be used in `isinstance`/`issubclass` calls and this syntax is more concise and readable, IMO.

Reply:
I believe it's allowed, but why this specific place should use a different style from all the other isinstance checks?

Reply:
It's the only place where multiple types are used with `isinstance`, hence the only place that has these changes.